kornia/testing/__init__.py

```diff
@@ -183,6 +183,26 @@ def assert_close(
 
 
 # Logger api
+def KORNIA_CHECK_SHAPE(x, shape: List[str]) -> None:
```

Comment:
awesome, i love it ! something I think important to check  is torchscript -- I had some headaches trying to "generalize" before the `KORNIA_CHECK` specially for the types.